### PR TITLE
added custom context menu support

### PR DESCRIPTION
**Pull request by omridevk**
# Context Menu
#### Supported Browsers:
- Internet Explorer 9-11
- Safari
- Chrome
- Firefox
  <hr>
  Example

Example#2
# 
#### [Demo](http://ec2-52-26-130-213.us-west-2.compute.amazonaws.com/mwEmbed/docs/ContextMenu)

To add item to the menu, simply give a value to the action.
# 

A custom context menu triggered whenever a user right click the player.

Using jquery.contextmenu library under M.I.T license.
## Usage:

``` javascript
kWidget.embed({
        targetId: "kaltura_player_1437157641899",
        wid: "_243342",
        uiconf_id: "20540612",
        entry_id: "1_23pqn2nu",
        flashvars: {
            "contextMenu": {
                "plugin" : true,
                "theme" : "normal-theme",
                "volumeUp" : "Volume Up",
                "volumeDown" : "Volume Down",
                "play" : "Play",
                "pause" : "Pause",
                "toggleFullscreen" : "Toggle Fullscreen"
            }
        }
})
```

The plugin require menu items, each menu item should correspond with an existing notification that can be sent to the player and a name that will set the menu item display name.

_TODO:_ 
- either offer a list of available events or either check if the event exist before setting the value.
- Add option to customize the look and feel of the menu.
- Add support for this plugin in the Universal Studio - In development in local CE installation.

### CODE REVIEW

File: modules/KalturaSupport/components/contextMenu/contextMenu.js [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+            showTooltip: true,
+            parent: "topBarContainer",
+            templatePath: 'components/contextMenu/contextMenu.tmpl.html',
+            theme: 'normal',
+            shortSeekTime: 5,
+            longSeekTime: 10,
+            volumePercentChange: 0.1,
+            volumeUp: null,
+            volumeDown: null,
+            pause: 'Pause',
```
Comment (itaykinnrot) [inline modules/KalturaSupport/components/contextMenu/contextMenu.js]: i18n

File: modules/KalturaSupport/components/contextMenu/contextMenu.js [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+        },
+        hideScreen: function(){
+            this._super();
+            if (this.getPlayer().getPlayerElement()) {
+                $( "#" + this.getPlayer().getPlayerElement().id ).removeClass( "blur" );
+                this.getPlayer().getPlayerPoster().removeClass( "blur" );
+            }
+        },
+        isSafeEnviornment: function () {
+            return !mw.isIpad() || ( mw.isIpad() && mw.getConfig('EmbedPlayer.EnableIpadHTMLControls') !== false );
```
Comment (itaykinnrot) [inline modules/KalturaSupport/components/contextMenu/contextMenu.js]: check for touch device

**Comment by itaykinnrot** [comment]
@omridevk please re-align to 2.35 _ change the php files to json (you'll see the other examples) and i'll land it.
thanks

**Comment by OrenMe** [comment]
closing due to inactive

**Pull request closed without merging**